from_dict/_from_dict.py

```diff
@@ -251,9 +252,17 @@ def from_dict(
         have an effect if constructed object has a __dict__.
     :param fd_global_ns: global namespace to help with handling of forward references encoded as string literals
     :param fd_local_ns: local namespace to help with handling of forward references encoded as string literals
+    :param fd_error_on_unknown:
+        Should an error be raised if additional arguments are supplied that are not used in constructor.
+        If this is True, fd_copy_unknown has to be set to False
```

Comment:
The other option is to have `fd_error_on_unknown` take precedent over `fd_copy_unknown`

Reply:
I think it's best for those to be mutually exclusive.

Can we say that we are raising an exception and also describe which exception we are raising?